packages/docs/page-config/extensions/unocss/code/unocss.config.js

```diff
@@ -0,0 +1,30 @@
+// unocss.config.*
+import { defineConfig, presetIcons, presetUno } from 'unocss'
+import presetTheme from 'unocss-preset-theme'
+import { colorsPreset, thresholdsPreset } from 'vuestic-ui'
+
+const convertNumbersToCssUnits = (obj, unit = 'px') =>
+  Object.entries(obj).reduce((acc, [key, value]) => ({
+    ...acc,
+    [key]: `${value}${unit}`,
+  }), {})
+
+export default defineConfig({
+  theme: {
+    colors: colorsPreset.light,
+    breakpoints: convertNumbersToCssUnits(thresholdsPreset),
+  },
+  presets: [
+    presetUno({
+      dark: 'media',
+    }),
+    presetIcons(),
+    presetTheme({
+      theme: {
+        dark: {
+          colors: colorsPreset.dark,
+        },
```

Comment:
```suggestion
        dark: {
          colors: colorsPreset.dark,
        },
        light: {
           colors: colorsPreset.light
        }
```

Seems quite logical to me. Not sure if it is correct.

Reply:
As I understood, preset was designed like that.